pyanaconda/modules/storage/partitioning/manual_partitioning.py

```diff
@@ -34,19 +32,25 @@
 class ManualPartitioningTask(NonInteractivePartitioningTask):
     """A task for the manual partitioning configuration."""
 
+    def __init__(self, storage, requests):
+        """Create a task.
+
+        :param storage: an instance of Blivet
+        :param requests: a list of requests
+        """
+        super().__init__(storage)
+        self._requests = list(requests)
```

Comment:
If it should be list (based on the doc string) why are you making a list from the `requests` variable?

Reply:
Fixed.